### PR TITLE
Licensing: Send the app url to plugin

### PR DESCRIPTION
**Pull request by leandro-deveikis**
Fixes https://github.com/grafana/grafana-enterprise/issues/1636

### CODE REVIEW

File: pkg/plugins/ifaces.go [hunk 1]
```diff
@@ -99,6 +99,8 @@ type Licensing interface {
 	Edition() string
 
 	Path() string
+
+	AppUrl() string
```
Comment (xnyo) [inline pkg/plugins/ifaces.go:103]: Nit: I think this should be `AppURL` instead?

Comment (leandro-deveikis) [reply pkg/plugins/ifaces.go]: yeah, I think so :)  fixed in 3ec2129ccc2acc61d7588919fe76a0a5b5f45253

**Review by wbrowne — APPROVED** [review]
Nit: Would be good to make sure all references to `Url` are made `URL` for consistency

**Review by xnyo — APPROVED** [review]
LGTM but I would also reccommend replacing Url with URL in the other instances like Will said :)

**Comment by leandro-deveikis** [comment]
Thanks @xnyo and @wbrowne I fixed the URLs on edd46d9fa96d73392fb27746307826dd852b10b8 (sorry I didnt do it on the previous commit)

**Review by wbrowne — APPROVED** [review]
Thanks for taking care ❤️

**Comment by mmandrus** [comment]
I see you already have approvals, but I would recommend changing the title according to the PR guidelines 👍

**Review by DanCech — APPROVED** [review]
LGTM

**Pull request merged**